components/dashboard/src/feedback-form/FeedbackComponent.tsx

```diff
@@ -54,12 +65,40 @@ function FeedbackComponent(props: { onClose: () => void; onSubmit: () => void; i
             </button>
         ));
     };
+
+    const minimisedFirstView = !selectedEmoji && !isFeedbackSubmitted;
+    const expandedWithTextView = selectedEmoji && !isFeedbackSubmitted;
+
     return (
         <>
-            <h3 className="mb-4">Send Feedback</h3>
-            {selectedEmoji ? (
+            {props.isModal && !isFeedbackSubmitted && <h3 className="mb-4">Send Feedback</h3>}
+            {minimisedFirstView && (
+                <div
+                    className={
+                        "flex flex-col justify-center px-6 border-t border-gray-200 dark:border-gray-800 " +
```

Comment:
**question**: Did we discuss anywhere the idea of reusing the existing feedback modal when a user clicks on one of the emojis? This could reuse the same component as is and save us a ton of vertical space, which is quite crucial in the workspace start page.

Reply:
Do you mean that instead of expanding in page, we would trigger the modal?
Let me loop in @jldec

Reply:
> Do you mean that instead of expanding in page, we would trigger the modal?

Yes.

> Good point re the vertical space, but in an error page, the user might want to keep the error and context always in sight, instead of blocked by a modal.

Fair point, just wanted to bring this up in case this has not been taken into consideration.